eclass/java-pkg-simple.eclass

```diff
@@ -506,6 +507,11 @@ java-pkg-simple_src_test() {
 	# create the target directory
 	mkdir -p ${classes} || die "Could not create target directory for testing"
 
+	# generated test classes should get generated into "generated-test" directory
+	if [[ -d generated-test ]]; then
+		cp -r generated-test/* "${classes}" || die "cannot copy generated test classes"
+	fi
+
```

Comment:
i think this feature should be documented in the function description

Reply:
Just thinking about a good explanation ...